custom_components/smartthinq_sensors/wideq/devices/dehumidifier.py

```diff
@@ -314,9 +319,36 @@ def water_tank_full(self):
             return None
         return self._update_feature(DehumidifierFeatures.WATER_TANK_FULL, value)
 
+    @property
+    def notification_light(self) -> bool | None:
+        """Return notification light status."""
+        try:
+            key = self._get_state_key(STATE_NOTIFICATION_LIGHT)
+        except:
+            key = None
+            _LOGGER.exception("LGE ThinQ dehumidifier - unable to get Notification Light status")
+        if key is None:
+            ntf_real_value = None
+            ntf_light_int_value = None
+            ntf_light_bool_val = None
```

Comment:
This will generate a lot of log exception for devices that do not support this feature. Please just return Nonw